tools/downloader/topologies.py

```diff
@@ -0,0 +1,81 @@
+import os
+import argparse
+import sys
+
+from pathlib import Path
+from argparse import Namespace
+
+import common
+import downloader
+import converter
+
+_cache = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', Path('.').resolve() / 'models'))
+_cache_dir = _cache / 'models_cache'
+
+def load_model(model_name, precision='FP32'):
```

Comment:
Could you add path as function argument?

Reply:
I'd like to suggest to keep this question for internal arch review process. Not hard to change but we just need to decide which version to choose.

But it's good idea to make it optional but let's check if there is a way to force method to have named arguments only so we won't meet a situation when order or set of arguments will be changed.

Reply:
@likholat, please add `output_dir` argument